tests/unit/handler.unitTest.ts

```diff
@@ -93,12 +93,14 @@ describe("The lambda function handler", () => {
                 };
 
                 // Stub out the actual functions
-                ActivityService.prototype.endActivity = jest.fn().mockResolvedValue({testResponse: 1234});
+                ActivityService.prototype.endActivity = jest.fn().mockResolvedValue({visitClosed: true});
                 let ctx: any = mockContext(opts);
                 const result = await handler(testEvent, ctx, () => { return; });
                 ctx.succeed(result);
                 ctx = null;
-                expect(result.statusCode).toEqual(204);
+                expect(result.statusCode).toEqual(200);
+                const expectedBody = JSON.parse(result.body)
+                expect(expectedBody.visitClosed).toBe(true);
```

Comment:
Same here. What is this checking? looks like it checks that the mock worked, rather than any logic